### PR TITLE
Walkthrough module

### PR DESCRIPTION
**Pull request by GeerSwitch**
Adds webpack, animation for deployment module.

### CODE REVIEW

File: _grow/gulp/tasks/styles.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-var gulp = require('gulp');
-var rename = require('gulp-rename');
-var plumber = require('gulp-plumber');
-var sass = require('gulp-sass');
-var sassGlob = require('gulp-sass-glob');
-var autoprefixer = require('gulp-autoprefixer');
+const config = require('../config');
+const gulp = require('gulp');
+const rename = require('gulp-rename');
+const plumber = require('gulp-plumber');
```
Comment (fertolg) [inline _grow/gulp/tasks/styles.js]: Doesn't seem like this is used anymore, maybe we can remove.

File: _grow/source/js/video-modal.js [hunk 2]
```diff
@@ -5,175 +5,166 @@
...
-     * Callback for clicking elsewhere on the document to close the modal.
-     * @todo(sgeer)
-     */
-    function checkModalClose(e) {
-      if (e.target.classList.contains('video-modal__backdrop')) {
-        closeModal(null, true);
-      }
+  /**
+   * Callback for clicking elsewhere on the document to close the modal.
+   * @todo(sgeer)
```
Comment (fertolg) [inline _grow/source/js/video-modal.js]: what's the todo here

File: _grow/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+  entry: './source/js/main.js',
```
Comment (fertolg) [inline _grow/webpack.config.js]: Just wondering why isn't this coming from the gulp/config.js constants file, we are piping webpack after using this file in gulp, but not sure if this config is doing something.

File: _grow/gulp/tasks/styles.js (hunk 1 above)
Comment (GeerSwitch) [reply _grow/gulp/tasks/styles.js]: Removing.

File: _grow/source/js/video-modal.js (hunk 2 above)
Comment (GeerSwitch) [reply _grow/source/js/video-modal.js]: Leftovers. Removed.

File: _grow/webpack.config.js (hunk 3 above)
Comment (GeerSwitch) [reply _grow/webpack.config.js]: Moved to config, gulp needs something in `src` (empty string here works as well).

**Pull request merged**